calcom/cal.com
# save additional inputs as json + view details of booking

## What does this PR do?

- Saves additional inputs as JSON data
- Prefills additional inputs when rescheduling 
- Fixes issues with showing additional inputs with `<br /><br/>`
- Adds possibility to click on a booking to see details of booking + cancel and reschedule 

Fixes #2681 
Fixes #2347 

## Type of change


## How should this be tested?

1. Book meeting of an event-type that has additional inputs
2. Reschedule meeting and see that additional inputs are prefilled 
   - Attendees can change additional inputs and additional notes 
   - Organizers can't change additional inputs and additional notes
3. Go to page where all bookings are listed 
4. Click on a booking 
5. See details of booking 

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
